ci_scripts/sync_target_database.py

```diff
@@ -140,18 +152,20 @@ def main(args: argparse.Namespace) -> int:
         body=args.pr_description,
     )
     try:
-        targets = MbedTargets().json_dump()
+        online_targets = MbedTargetsOnline()
+        news_file_name = f"{datetime.date.today().strftime('%Y%m%d')}"
         if TARGET_DATABASE_PATH.exists():
-            diff_objects = diff_objects_from_json_str(targets, TARGET_DATABASE_PATH.read_text())
-            if not diff_objects:
+            offline_targets = MbedTargetsOffline()
+            added, removed = get_boards_added_or_removed(offline_targets, online_targets)
+            if not len(added) and not len(removed):
```

Comment:
```suggestion
            if not (added or removed)
```

No need for `len()`.

Reply:
This has a syntax error, you missed the colon. Fixed and applied anyway :)

Reply:
I realized that too, sorry. 😅